tests/npc_talk_test.cpp

```diff
@@ -24,37 +25,47 @@ const efftype_id effect_infected( "infected" );
 static const trait_id trait_PROF_FED( "PROF_FED" );
 static const trait_id trait_PROF_SWAT( "PROF_SWAT" );
 
-npc create_test_talker()
+npc *create_test_talker()
```

Comment:
This should return a reference (the pointer it currently returns is never null anyway). Then you could store the result as reference in the calling function below and you would not need to change all the lines above from `.` to `->`.

Reply:
Fixed, thanks.
Still doesn't resolve Travis failing the tests that work fine on my test box.

Reply:
Test harness has STATIC_NPCS set to off. Fixed the test harness.